CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # authn.io ChangeLog
 
+## 2.1.0 - 2022-03-xx
```

Comment:
With all the major releases in bedrock space, I cannot imagine that this would be considered a minor release if we were already in an infrastructure as code environment.  Please make this a 3.0.0 release.

Reply:
This is a major release simply on the basis that we've changed the core config and have dockerized it and so on.